media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild

```diff
@@ -61,13 +67,6 @@ pkg_preinst() {
 pkg_postinst() {
 	fdo-mime_desktop_database_update
 	gnome2_icon_cache_update
-
```

Comment:
Now that's a bit confusing commit set. I'll move this removal to the previous commit to make it a little less confusing.